massa-network-worker/src/node_worker.rs

```diff
@@ -27,14 +31,21 @@ use tokio::{
 };
 use tracing::{debug, trace, warn};
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum ToSend {
+    Msg(Message),
```

Comment:
Maybe explicitely ignore the warning ? To show in the code that there is indeed a warning ?

Reply:
Why would you want to show in the code that there is warning? It's not actually a problem as far as I can tell, and leaving something in the code makes it look a bit like a TODO, and I don't think we should care about this at all unless there is a problem. 

For example here I don't think we need to box the `Message` variant just to make the warning go away, because, why would we? I actually introduced a smaller variant, previously everything was a `Message`, so this would appear not to be a possible loss in performance.

A box would mean an additional pointer and heap allocation(I think), which actually might degrade performance more. 

And actually both of these considerations in my opinion do not matter unless this particular thing were to show up on a performance test.

I am not a big fan of clippy, since it appears to often makes things look like problems when there aren't actually any.

Reply:
Actually the compiler will align the enum with the sizeof Message even if it gets just a BlockId, here is globally the purpose of that warning. Would you like a PR with a proposal? :slightly_smiling_face: 

Here is a video from Rust explaining why that is a performance wickness ![on youtube](https://www.youtube.com/watch?v=rAF8mLI0naQ). @yvan-sraka used to broadcast that video some month ago in the team.

Reply:
it's when she speaks about the stack size at perhaps 23 minutes

Reply:
@adrien-zinger Thanks for the video.

> Actually the compiler will align the enum with the sizeof Message even if it gets just a BlockId

How can that be a performance regression, if previously all variants were `Message`?

On the other hand, boxing `Message` would mean a extra pointer access that previously wasn't there.

Also, this is an isolated case(not "all errors across the application" like in the video), so in my opinion it's irrelevant, unless it were to show up somewhere on a performance test, in which case the fix would be easy.